ligo/Makefile

```diff
@@ -0,0 +1,32 @@
+# SPDX-FileCopyrightText: 2020 TQ Tezos
+# SPDX-License-Identifier: MIT
+
+# Ligo executable
+LIGO ?= ligo
+
+# Compile code
+BUILD = $(LIGO) compile-contract --syntax cameligo
```

Comment:
What syntax are we using? I recall this message in our chat:
> Regrading ligo syntax, let's pick Pascal. The tutorial is written on this dialect so using it should be simpler

Reply:
Eventually Kirill Kuvshinov opened my eyes and now I see that PascalLigo is not that nice, better to use camelligo.

Reply:
IIRC it was your lowest priority, but ok, I don't have a preference anyway. It means we won't be able to copy-paste FA2 implementation from Stablecoin, but ok, we will port it to a different LIGO syntax, shouldn't be hard.